operations/configuration_management/Specifying_Hosts_and_Groups.md

```diff
@@ -0,0 +1,15 @@
+# Specifying Hosts and Groups
+
+When using CFS, there are many steps where you may need to specify the hosts CFS should configure.  This can be done by specifying individual hosts, or groups of hosts.  However there are several places where a user may have to provide this information, particularly groups, and depending on where this information is provided, the behavior can change greatly.
```

Comment:
```suggestion
When using the Configuration Framework Service (CFS), there are many steps where users may need to specify the hosts that CFS should configure. This can be done by specifying individual hosts, or groups of hosts. There are several places where a user may need to provide this information, particularly groups, and depending on where this information is provided, the behavior can change greatly.
```